pkg/logs/client/destination.go

```diff
@@ -6,7 +6,18 @@
 package client
 
 import (
+	"expvar"
 	"net"
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/metrics"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// FIXME: Changed chanSize to a constant once we refactor packages
+const (
+	chanSize      = 100
```

Comment:
I am not a huge fan of this, it should be injected.

Reply:
I'd be happy to change it once #2504 is merged